src/builtins.ts

```diff
@@ -3592,7 +3592,12 @@ export function compileCall(
     case BuiltinSymbols.idof: {
       let type = evaluateConstantType(compiler, typeArguments, operands, reportNode);
       compiler.currentType = Type.u32;
-      if (!type) return module.unreachable();
+      if (!type || checkArgsOptional(operands, 0, 0, reportNode, compiler)) return module.unreachable();
```

Comment:
The arguments check should already be covered by `evaluateConstantType` above

Reply:
Okay. I'll remove the additional check.

Reply:
This was removed